plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1326,6 +1326,23 @@ blockly.block.set_event_result.line1=Set result of global trigger event to
 blockly.block.set_event_result.line2=Works only with global triggers with a result
 blockly.block.call_procedure=Call procedure
 blockly.block.procedure_retval=Call procedure and get return value
```

Comment:
Is the key blockly.block.procedure_retval still needed?

Reply:
Checking https://github.com/MCreator/MCreator/pull/4721/commits/d63b463115df4c7f1b184760f260aacc578b362a, it seems the fallback is not really working as intended

Reply:
But as explained in my second comment, I don't think fallback is working as intended, given you needed to specify https://github.com/MCreator/MCreator/pull/4721/commits/d63b463115df4c7f1b184760f260aacc578b362a for tests to pass.

> Checking L10N.t method, for those cases null will not be returned currently so this (L10N.t) may need to be modified

Reply:
> Checking L10N.t method, for those cases null will not be returned currently so this may need to be modified

Seems to be only for tests — removing one of the variable-specific keys and opening MCreator works as intended